src/notebook/index.css

```diff
@@ -37,67 +37,87 @@
 }
 
 
-.jp-MarkdownCell-renderer.p-Widget {
+.jp-MarkdownCell-content.p-Widget {
+  padding-left: 5.6px;
```

Comment:
I was just matching what I saw in the notebook, no particular reason. I can change it if you'd like.